vocab/properties.json

```diff
@@ -840,7 +828,7 @@
         ]
     },
     "hash": {
-        "description": null,
+        "description": "Function that converts original data into a single value.",
```

Comment:
also refers to the resulting single value. ("I used a hash function to calculate the hash of the file")

Reply:
I think we decided to use "digest" (hash digest) for the resulting single value and use "hash" for referring to both (function + digest)? Any idea how to solve this without confusing people too much?

Reply:
And you are right. Currently the description also only covers the "function". How about:
"Term used for the process of converting any data into a single value. Often also directly refers to the resulting single value."